### PR TITLE
Invoking python3 installation for PDDF 2.0 PAI package

### PR DESCRIPTION
**Pull request by fk410167**
#### Why I did it
2.0 Platform APIs python3 wheel package should get installed on the HOST as well as PMON docker

#### How I did it
Adjusting the PDDF utility with python3 commands

#### How to verify it
Check for the python3 package installation on a PDDF supported platform

```
root@sonic:/home/admin#
root@sonic:/home/admin# pip3 show sonic-platform
Name: sonic-platform
Version: 1.0
Summary: SONiC platform API implementation on Accton Platforms
Home-page: https://github.com/Azure/sonic-buildimage
Author: SONiC Team
Author-email: linuxnetdev@microsoft.com
License: Apache 2.0
Location: /usr/local/lib/python3.7/dist-packages
Requires:
Required-by:
root@sonic:/home/admin# python3
Python 3.7.3 (default, Jul 25 2020, 13:03:44)
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sonic_platform
>>> ch = sonic_platform.platform.Platform().get_chassis()
>>>
```

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

**Comment by fk410167** [comment]
retest please

**Comment by fk410167** [comment]
retest this please

**Comment by fk410167** [comment]
retest this please

**Comment by fk410167** [comment]
retest this please

**Comment by fk410167** [comment]
retest this please

**Comment by fk410167** [comment]
retest this please

**Comment by fk410167** [comment]
@jleveque 
Can you review and merge this?

**Comment by fk410167** [comment]
retest please

**Comment by fk410167** [comment]
retest this please

**Comment by fk410167** [comment]
retest this please

File: platform/pddf/i2c/utils/pddf_util.py [hunk 1]
```diff
@@ -486,60 +490,60 @@ def do_switch_pddf():
...
         if not status:
             # Need to remove this whl module
-            status, output = log_os_system("docker exec -it pmon pip uninstall sonic-platform -y", 1)
+            status, output = log_os_system("docker exec -it pmon pip3 uninstall sonic-platform -y", 1)
             if not status:
-                print "Successfully uninstalled the native sonic-platform whl pkg from pmon container"
+                print("Successfully uninstalled the native sonic-platform whl pkg from pmon container")
             else:
-                print "Error: Unable to uninstall the sonic-platform whl pkg from pmon container.\
+                print("Error: Unable to uninstall the sonic-platform whl pkg from pmon container.\)
```
Comment (jleveque) [inline platform/pddf/i2c/utils/pddf_util.py:514]: LGTM caught this syntax error. Closing parenthesis should be placed at the end of the next line and the backslash can be removed.

**Review by jleveque — CHANGES_REQUESTED** [review]

**Review by jleveque — APPROVED** [review]

**Comment by fk410167** [comment]
@jleveque 

Can you merge this please?

**Pull request merged**